### ISSUE TITLE
Could not initialize class org.bytedeco.javacpp.caffe$LayerParameter

### ISSUE THREAD

**Issue by formaten** [bytedeco/javacpp-presets#126]
i have downloaded the source code of javacpp-presets(1.2-SNAPSHOT)  and built it on my Macbook Air(OSX 10.11.2 and JDK 8),

but when i put the jar packages(caffe.jar,caffe-macosx-x86_64.jar,opencv.jar,opencv-macosx-x86_64.jar and javacpp.jar) to my helloworld project,

i got the some errors:
```
Error loading class org/bytedeco/javacpp/caffe$LayerParameter.
Error loading class org/bytedeco/javacpp/caffe$LayerParameter.
Exception in thread "main" java.lang.NoClassDefFoundError: Could not initialize class org.bytedeco.javacpp.caffe$LayerParameter
	at java.lang.ClassLoader$NativeLibrary.load(Native Method)
	at java.lang.ClassLoader.loadLibrary0(ClassLoader.java:1938)
	at java.lang.ClassLoader.loadLibrary(ClassLoader.java:1821)
	at java.lang.Runtime.load0(Runtime.java:809)
	at java.lang.System.load(System.java:1086)
	at org.bytedeco.javacpp.Loader.loadLibrary(Loader.java:595)
	at org.bytedeco.javacpp.Loader.load(Loader.java:453)
	at org.bytedeco.javacpp.Loader.load(Loader.java:396)
	at org.bytedeco.javacpp.caffe.<clinit>(caffe.java:16)
	at java.lang.Class.forName0(Native Method)
	at java.lang.Class.forName(Class.java:348)
	at org.bytedeco.javacpp.Loader.load(Loader.java:428)
	at org.bytedeco.javacpp.Loader.load(Loader.java:396)
	at org.bytedeco.javacpp.caffe$SolverParameter.<clinit>(caffe.java:2084)
	at Test.main(Test.java:431)
```
how can i solve it?Thanks!

**Comment by saudet** [comment]
Could you post a minimal version of your `Test.java` that produces this error? Thanks!

**Comment by saudet** [comment]
Also, could you give me the content of the caffe-macosx-x86_64.jar and opencv-macosx-x86_64.jar files? Maybe there's something missing there for some reason.

**Comment by formaten** [comment]
this is my helloworld project.




[Caffe4Java.zip](https://github.com/bytedeco/javacpp-presets/files/63569/Caffe4Java.zip)

**Comment by formaten** [comment]
Hi,Saudet!
Have you found any problems in this project?
:-)

**Comment by saudet** [comment]
Ah, well, the link doesn't work. In any case, I'll be running a build under Mac OS X myself when I have some time this weekend, to make sure it works. 

BTW, this is something that could be automated if we had a CI server somewhere. If you'd like to do something for this project, that would be a priority. :)

**Comment by formaten** [comment]
ok,thanks!

I haved reuploaded the project to my online-drive,you can download by the following link:
http://vdisk.weibo.com/s/GnJqPzdn9QWo
:-)

**Comment by saudet** [comment]
I've been able to reproduce the problem. I've fixed an issue with the `Loader` in https://github.com/bytedeco/javacpp/commit/ffab0a367c6e57455f8be3117cd7a75167d50000 that seems to help with loading recent versions of Caffe. There is still an error message that pops up, but the Caffe seems to be functional, so I believe it's more of a warning message. So, please let me know if everything works now, thanks!

**Comment by formaten** [comment]
I've retried your new code of javacpp in my MacBook, but it shows the same errors and not work(the application will exit automatically).

And then, i also tried in Ubuntu 14.10(64 bit),it works fine for me.

**Comment by saudet** [comment]
Ok, the latest commit above should fix everything! Let me know if you still have some issues though.

**Comment by saudet** [comment]
Fix included in the release of version 1.2, thanks for reporting!
